components/automate-ui/package.json

```diff
@@ -73,7 +72,7 @@
     "ajv": "^6.5.3",
     "angular2-template-loader": "^0.6.2",
     "axe-webdriverjs": "^2.0.1",
-    "codelyzer": "^4.4.4",
+    "codelyzer": "^4.5.0",
```

Comment:
The changes to these deps feels very minor, it might be worth putting them in their own PR to make this one smaller.